### PR TITLE
feat(Avatar): add size prop

### PR DESCRIPTION
**Pull request by christian-dinh**
### Overview

Add a `size` prop to the `Avatar` component

### PR Checklist

- [x] Related to [designs](https://www.figma.com/file/SwfwQZmV4jtK68QaCa89h3/CC-Docs-v2---User-Recognition?node-id=452%3A57785)
- [x] Related to JIRA ticket: [REACH-1265](https://codecademy.atlassian.net/browse/REACH-1265)
- [x] I have run this code to verify it works
- [x] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://612fe62842ead824fe08fdfd--gamut-preview.netlify.app>https://612fe62842ead824fe08fdfd--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/612fe62842ead824fe08fdfd>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.5.54-alpha.207035.0<br/>@codecademy/gamut-labs@19.24.0-alpha.207035.0<br/>@codecademy/styleguide@43.10.11-alpha.207035.0<br/></pre>

**Review by JoshuaKGoldberg — DISMISSED** [review]

File: packages/gamut-labs/src/Avatar/index.tsx [hunk 1]
```diff
@@ -63,6 +63,11 @@ export type AvatarBaseProps = {
    */
   disableDropshadow?: boolean;
 
+  /**
+   * width and height of the Avatar in px
+   */
+  size?: number;
```
[the remark this answers is not in the document]
Comment (christian-dinh) [reply packages/gamut-labs/src/Avatar/index.tsx:69]: yeah I can do `small = 32px` and `medium = 118px` for now if that sounds ok, or maybe `medium` should be `large`

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://612ff97502182e0dcbb8ce03--gamut-preview.netlify.app>https://612ff97502182e0dcbb8ce03--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/612ff97502182e0dcbb8ce03>Deploy Logs</a></p>

File: packages/gamut-labs/src/Avatar/index.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (JoshuaKGoldberg) [reply packages/gamut-labs/src/Avatar/index.tsx]: I'll defer to @codecaaron and @dreamwasp on that one. No particular preference from me 😄

[the remark this answers is not in the document]
Comment (codecaaron) [reply packages/gamut-labs/src/Avatar/index.tsx]: 👍  to `small` and `medium`

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.5.56-alpha.ead40e.0<br/>@codecademy/gamut-labs@19.24.0-alpha.ead40e.0<br/>@codecademy/styleguide@43.11.2-alpha.ead40e.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6131011133e8d6e2faecf081--gamut-preview.netlify.app>https://6131011133e8d6e2faecf081--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6131011133e8d6e2faecf081>Deploy Logs</a></p>

**Review by dreamwasp — APPROVED** [review]
🙏

**Pull request merged**